pkg/csi/snapshot_restore_ops_test.go

```diff
@@ -34,42 +35,42 @@ func (s *CSITestSuite) TestGetDriverNameFromUVSC(c *C) {
 		{
 			vsc: unstructured.Unstructured{
 				Object: map[string]interface{}{
-					VolSnapClassAlphaDriverKey: "p2",
+					common.VolSnapClassAlphaDriverKey: "p2",
 				},
 			},
-			version: alphaVersion,
+			version: common.SnapshotAlphaVersion,
 			expOut:  "p2",
 		},
 		{
 			vsc: unstructured.Unstructured{
 				Object: map[string]interface{}{},
 			},
-			version: alphaVersion,
+			version: common.SnapshotAlphaVersion,
 			expOut:  "",
 		},
 		{
 			vsc: unstructured.Unstructured{
 				Object: map[string]interface{}{
-					VolSnapClassBetaDriverKey: "p2",
+					common.VolSnapClassBetaDriverKey: "p2",
 				},
 			},
-			version: betaVersion,
+			version: common.SnapshotBetaVersion,
 			expOut:  "p2",
 		},
 		{
 			vsc: unstructured.Unstructured{
 				Object: map[string]interface{}{},
 			},
-			version: betaVersion,
+			version: common.SnapshotBetaVersion,
 			expOut:  "",
 		},
 		{
 			vsc: unstructured.Unstructured{
 				Object: map[string]interface{}{
-					VolSnapClassBetaDriverKey: map[string]string{},
+					common.VolSnapClassBetaDriverKey: map[string]string{},
 				},
 			},
-			version: betaVersion,
+			version: common.SnapshotBetaVersion,
```

Comment:
Should we add tests in this table for `common.SnapshotStableVersion` ?

Reply:
Yea, will add them. Keeping me honest